### PR TITLE
Chore about shell

### PR DESCRIPTION
**Pull request by kaikai1024**
### Description of the Change

* Refactor util about timeout
* Remove useless script(do not maintain)
* `enable_tls` as a param of cita basic test

### Applicable Issues

#604

### CODE REVIEW

**Comment by ouwenkg** [comment]
CI failed.

**Comment by kaikai1024** [comment]
> CI failed.

A little jobe to do.

File: tests/integrate_test/cita_basic.sh [hunk 1]
```diff
@@ -1,117 +1,97 @@
...
-################################################################################
-echo -n "6) start node2, check height growth  ...  "
-bin/cita bebop start node/2 trace > /dev/null &
-timeout=$(check_height_growth_normal 0 60) || (echo "FAILED"
-                                               echo "error msg: ${timeout}"
-                                               exit 1)
-echo "${timeout}s DONE"
+    echo -n "6) start node2, check height growth  ...  "
+    bin/cita bebop start node/2 trace > /dev/null &
+    timeout=$(check_height_growth_normal 0 60) || (echo "FAILED"
```
Comment (leeyr338) [inline tests/integrate_test/cita_basic.sh]: Is 60s enough? Recommend 90s.

Comment (kaikai1024) [reply tests/integrate_test/cita_basic.sh]: i don't change this value.
Just add a main function and make the tls test as a part of this script.

Comment (leeyr338) [reply tests/integrate_test/cita_basic.sh]: Okay!

File: tests/integrate_test/util.sh [hunk 2]
```diff
@@ -79,210 +60,188 @@ get_peer_count(){
...
+                expected count: ${expected_count}"
             return 1
         fi
         sleep 1
     done
-    return 1
 }
 
-check_height_growth_normal () {
+check_height_growth_normal() {
```
Comment (kaikai1024) [inline tests/integrate_test/util.sh]: check this

File: tests/integrate_test/util.sh [hunk 3]
```diff
@@ -79,210 +60,188 @@ get_peer_count(){
...
+    local id=$1
+    local timeout=$2
+    local old
+    local now
+
+    if old=$(get_height "${id}"); then
+        echo "old height: ${old}"
+        start=$(date +%s)
+        while true; do
+            if check_height_growth "${id}" "${old}"; then
```
Comment (leeyr338) [inline tests/integrate_test/util.sh]: 这里应该有问题。

`check_height_growth "${id}" "${old}"`

这里获得的应该是里面的 echo 值，而不是 return 结果。另外，如果这进而获得的是 return 结果，那么这里应该要取反，因为在 `check_height_growth` return 0 表示获得正确结果。对吗？

Comment (kaikai1024) [reply tests/integrate_test/util.sh]: https://stackoverflow.com/questions/8742783/returning-value-from-called-function-in-a-shell-script

I will fix the debug info.

> 因为在 check_height_growth return 0 表示获得正确结果

but why use the 

> 那么这里应该要取反，

Why?

Comment (kaikai1024) [reply tests/integrate_test/util.sh]:
```shell
check() {
    echo "test"
    return 0
    # return 1
}

if check; then
    echo "a"
else
    echo "b"
fi
```

run this.

**Pull request merged**